packages/app-desktop/gui/NoteEditor/utils/contextMenu.ts

```diff
@@ -53,17 +55,52 @@ function handleCopyToClipboard(options: ContextMenuOptions) {
 	}
 }
 
-export function menuItems(): ContextMenuItems {
+export async function openItemById(options: any, dispatch: Function) {
```

Comment:
Looking at the code, you only need an ID and an optional hash, so pass this to the function explicitely: `itemId: string, dispatch:Function, hash:string = ''` Please do not use "any" as it disables type checking.